bot/cogs/reddit.py

```diff
@@ -41,12 +43,15 @@ async def reddit_embed(subreddit: str, randompost: RedditAPI.submission) -> Embe
     return embed
 
 
-with open("bot/assets/subreddit.json", "r") as f:
+with open(f"bot{os.path.sep}assets{os.path.sep}subreddit.json", "r") as f:
```

Comment:
The same as before, you can use `join` here instead.

Reply:
Done